test/function_tests/results/install-results.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The results folder should be added to the gitignore because we don't want specific test results in the git history